build/deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM busybox
+FROM buildpack-deps:jessie-curl
```

Comment:
I ran across alpine but assumed that a musl-based image at 5mb wouldn't have glibc, which is what we need with our recent choices. Am I wrong there?